api_app/analyzers_manager/observable_analyzers/spyse.py

```diff
@@ -0,0 +1,57 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+import re
+from api_app.exceptions import AnalyzerRunException
+from api_app.analyzers_manager import classes
+
+from tests.mock_utils import if_mock_connections, patch, MockResponse
+
+
+class Spyse(classes.ObservableAnalyzer):
+    base_url: str = "https://api.spyse.com/v4/data/"
+
+    def set_params(self, params):
+        self.__api_key = self._secrets["api_key_name"]
+
+    def run(self):
+        params = {"key": self.__api_key}
```

Comment:
This is incorrect. The [docs](https://spyse.com/api) say it requires an header instead.

`Authorization: Bearer 776b9620-e645-4f80-81e0-e45eebdade06`